src/dev/ContentList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, ReactElement, StrictMode } from 'react';
+import ReactDOM from 'react-dom';
+
+import { Provider as ClientProvider } from '@aiera/client-sdk/api/client';
```

Comment:
I actually refactored all this into a single `Provider` yesterday so if you rebase onto master you can just grab the new one instead, should be super easy.